src/pages/patientView/PatientViewPage.tsx

```diff
@@ -447,6 +464,23 @@ export default class PatientViewPage extends React.Component<
         }
     }
 
+    @autobind
+    private onTableRowClick(d: Mutation[]) {
```

Comment:
we should be specific about which table this pertains to.   i.e. onMutationTableRowClick

Reply:
true for below as well

Reply:
Fixed